chain/gen/genesis/t01_init.go

```diff
@@ -30,6 +32,16 @@ func SetupInitActor(bs bstore.Blockstore, netname string, initialActors []genesi
 	amap := hamt.NewNode(cst, hamt.UseTreeBitWidth(5)) // TODO: use spec adt map
 
 	for i, a := range initialActors {
+		if a.Type == genesis.TMultisig {
+			addr, _ := address.NewActorAddress(a.Meta)
+			fmt.Printf("init set %s t0%d\n", addr, AccountStart+uint64(i))
+
+			if err := amap.Set(context.TODO(), string(addr.Bytes()), AccountStart+uint64(i)); err != nil {
+				return nil, err
+			}
```

Comment:
That's not needed, is it?

Reply:
mmmm, no. i think we can get away with just an ID address for all of these